server/protocols/jmap/src/test/java/org/apache/james/jmap/model/MessageFactoryTest.java

```diff
@@ -164,6 +165,11 @@ public void headersShouldBeSetIntoMessage() throws Exception {
                 .preview("(Empty)")
                 .textBody(Optional.of(""))
                 .htmlBody(Optional.empty())
+                .isAnswered(false)
```

Comment:
Here, I should either use **is\*** _or_ **keywords**.

You should fail if you use both. Otherwise, how can we ensure the state is consistent?

Also internally, you message should only have **keywords** stored as a field, and **is\*** properties should be computed from **keywords**. 

I already made this remark 3 time...